content/cloud/deployment-concepts.md

```diff
@@ -8,54 +8,46 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/deployment-c
+## The Deployment Process Explained
 
-Following packaging, the bindle will be uploaded to the Fermyon Cloud. As soon as a bindle has been uploaded, it cannot be modified or deleted. This is to preserve the integrity of the immutability of bindles.
+When you run the `spin cloud deploy` command, the following happens:
 
-### 2. Create or Upgrade an Application
+1. Packaging the application and uploading it to the Fermyon Cloud
+  - The first step in deploying an application is to package all the files using the OCI specification. The Cloud Plugin will create individual layers for the files that make up the application, to optimize changes to applications on subsequent packaging and deployment.
+  - Following packaging, the image will be uploaded to an OCI compatible registry in the Fermyon Cloud.
+2. Creating or upgrading an application
+  - An application is upgraded if the application name, as defined in `spin.toml`, already exists in your Fermyon Cloud account.
+  - Before serving traffic to the application, the deployment process checks for the application health endpoint and finishes once the application is concluded to be healthy by the cloud. The application health point is an integral part of the Fermyon Cloud, which does reserve the HTTP route `/.well-known/spin/health`, which will not be routed to your Spin components.
+  - If an upgrade takes place, the failover from the old to the new deployment takes a short amount of time, during which you can observe replies from both deployments (version).
 
-The next step in the deployment process is to create or upgrade the application.
+## Deprecating Bindle - An Aggregate Object Storage System
 
-An application in the Fermyon Cloud can have multiple revisions, which are tied to channels. These concepts are derived from [Hippo](https://docs.hippofactory.dev/) an open-source Platform as a Service (PaaS) for WebAssembly. As you deploy your application both an application, a channel and a revision will be created in the Fermyon Cloud.
+With the release of the [Spin Cloud Plugin](https://developer.fermyon.com/cloud/cloud-command-reference) [v0.3.0](https://github.com/fermyon/cloud-plugin/releases/tag/v0.3.0), the default deployment mechanism is OCI, not Bindle. The Fermyon Cloud still supports using [Bindle](https://github.com/deislabs/bindle) to package and distribute Spin applications. You can use a Cloud Plugin version [v0.2.0](https://github.com/fermyon/cloud-plugin/releases/tag/v0.2.0) to keep using Bindle, however, it's not guaranteed that all features of the version of the plugin will be compatible with other operations in the Cloud.
```

Comment:
I still feel that if we are going to say "you can use 0.2.0 to keep using Bindle", we should say something about when you might need to do this, i.e. almost never.  _grin_

Reply:
Agreed here with @itowlson; also seeing as we intend to fully deprecate use of the Bindle server in Fermyon Cloud in the near future.  Maybe we consider removing the latter sentence entirely?  Or a generalized note to 'reach out' if Bindle-based app deploy is somehow a strict need?

Reply:
Add a note to say that it's highly recommended to move to the latest Cloud plugin.